test.js

```diff
@@ -0,0 +1,70 @@
+var tests = {
+
+    	numbers: [1, 5, 66, 224, 6, 22, 4, 1, 6, 8, 9, 86, 3, 4, 32, 64, 10],
+
+    	/*
+	Given an integer array of size N, compute the sum of all even numbers in this array
+	*/
+	firstTest: function () {
+    		var array = this.numbers.slice();
+    		var result = 0;
+    	
+		for (var i = 0; i < array.length; i++) {
+			if (array[i]%2 == 0) {
+		  		result += array[i];
+		  	}
+		}
+    		return result;
+    	},
+
+    	/*
+	Given an integer array of size N, identify the largest 2 numbers in this array. Do not sort the array.
+	*/
+	secondTest: function () { 
+	 	var array = this.numbers.slice();
+	    	//get the biggest number
+		var biggestNr = Math.max.apply(Math, array);
+			
+		//remove the biggest number from original array
+		var index = array.indexOf(biggestNr);
+		array.splice(index, 1);
+	
+		//get the second biggest number
+		var secondNr = Math.max.apply(Math, array);
+		
+		return biggestNr + ' & ' + secondNr;
+	},
+
+    	/*
+	Given an array of numbers having size N in which every number is between 1 and N, determine 
+	if there are any duplicates in it. You are allowed to destroy the array if you like. 
+	Do not sort the array or use bit vectors. Try to work within the array without using any other 
+	temporary data structures. Do not use 2 nesting for loops since the time complexity would be high. 
+	Try to find a solution by executing just one loop. Hint: manipulate the elements of the same array
+	as you loop through.
+	*/
+	thirdTest: function () {
+	    	var array = this.numbers.slice();
```

Comment:
The number array that you give as input is not correct

The problem states: `'Given an array of numbers having size N in which every number is between 1 and N` your array contains the number 224 that means it should have at least 224 elements.